src/main/flight/pid.c

```diff
@@ -389,7 +389,11 @@ static FAST_RAM_ZERO_INIT bool itermRotation;
 static FAST_RAM_ZERO_INIT bool smartFeedforward;
 #endif
 #if defined(USE_ABSOLUTE_CONTROL)
+#ifdef UNIT_TEST
+float axisError[XYZ_AXIS_COUNT];
+#else
 static FAST_RAM_ZERO_INIT float axisError[XYZ_AXIS_COUNT];
```

Comment:
I think the define should be removed and STATIC_UNIT_TESTED should be used or?

Reply:
But I will see how these FAST_RAM things also can be implemented in some UNIT_TEST_* macro to make it better

Reply:
I think `STATIC_UNIT_TESTED` should work fine with `FAST_RAM_ZERO_INIT`, and in effect it will yield the same as the `#if` that you are using.

Reply:
Why this got marked as resolved? this still is wrong and should get changed as in the first message

STATIC_UNIT_TESTED FAST_RAM_ZERO_INIT float axisError[XYZ_AXIS_COUNT];

this ifdef UNIT_TEST is just dirty and should not be used in any possible case

Reply:
@wind0r @mikeller 
To me personally this really becomes a bit unreadable and confusing. I am not sure if we need to go to this path.
 `STATIC_UNIT_TESTED FAST_RAM_ZERO_INIT float axisError[XYZ_AXIS_COUNT];` 

In my syntax a new programmer immediatelly knows what is going on

Reply:
I'm your "syntax" you introduce unnecessary duplication which new programmers are advised against amongst the first fails.

Reply:
Ok let's use 70 lines to create a variable. I guess we save amount of lines now ;)

Reply:
If you don't see the difference between multiple sources of truth and line/symbol count, I give up. Bye.

Reply:
@DieHertz @wind0r Your change was incorporated immediatelly after your comment. What else is the issue here now?

Reply:
@borisbstyle oops sorry. i over read the "else" since the message was in this thread and not in the pr i thought it is still about this line and not the pr over all.

overall i think this is good to go. Personal i would hope that the changed to pid.h could be done in a different way but i currently do not know how.

Reply:
@wind0r Yes I was convinced after your comments and changed it immediatelly. No reason for further discussion ;)

I think in pid.h it is not bad to do it like this, since otherwise the functions aren't defined anyway in pid.h.